ldk/node/src/hostClients/stoppables.ts

```diff
@@ -1,15 +1,11 @@
 /**
  * The StreamListener type is a function provided to start a stream. It accepts two parameters and returns nothing.
  *
- * @typeParam T - The type of the messages sent.
```

Comment:
Please add this back in; the `typeParam` tag is consumed by TypeDoc. If it's complaining that it doesn't recognize the tag I think we can add that in the ESLint config.

Reply:
Gotcha - sorry I already merged this in before you commented! I see where the `@template` tag [is supported](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template), so I'll create a separate PR with that. I'm testing it now, and that seems to pass ESLint. Sound good?